openedx/features/caliper_tracking/tests/expected/edx.forum.thread.created.json

```diff
@@ -0,0 +1,59 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Posted",
+  "actor": {
+    "id": "edx.devstack.lms:18000/u/test",
```

Comment:
Should be replaced with http://localhost:18000

Reply:
done.

Reply:
@arooshaarif kindly review it agian.